py/picca/fitter2/data.py

```diff
@@ -133,6 +128,48 @@ def __init__(self,dic_init):
         if 'standard asymmetry' in dic_init['model']:
             self.xi_asy_model = partial(getattr(xi, dic_init['model']['standard asymmetry']), name=self.name)
 
+        self.bb = {}
+        self.bb['pre-add'] = []
+        self.bb['pos-add'] = []
+        self.bb['pre-mul'] = []
+        self.bb['pos-mul'] = []
+        if 'broadband' in dic_init:
+            for ibb,dic_bb in enumerate(dic_init['broadband']):
+                deg_r_min = dic_bb['deg_r_min']
+                deg_r_max = dic_bb['deg_r_max']
+                ddeg_r = dic_bb['ddeg_r']
+
+                deg_mu_min = dic_bb['deg_mu_min']
+                deg_mu_max = dic_bb['deg_mu_max']
+                ddeg_mu = dic_bb['ddeg_mu']
+    
+                name = 'BB-{}-{} {} {} {}'.format(self.name,
+                        ibb,dic_bb['type'],dic_bb['pre'],dic_bb['rp_rt'])
+
+                bb_pars = {'{} ({},{})'.format(name,i,j):0\
+                    for i in range(deg_r_min,deg_r_max+1,ddeg_r)\
+                        for j in range(deg_mu_min, deg_mu_max+1, ddeg_mu)}
+        
+                for k,v in bb_pars.items():
+                    dic_init['parameters']['values'][k]=v
+                    dic_init['parameters']['errors']['error_'+k]=0.01
```

Comment:
`dic_init['parameters']['errors'] = {'error_'+k:0.01 for k in bb_pars.keys()}` is more beautiful, but as you want.

Reply:
agreed, I'll make the change

Reply:
actually what you propose doesn't work. Your solution would override the `dic_init['parameters']['errors']` dictionary. What I'm doing here is adding new `k,v` pairs.